src/components/multi-action-button/multi-action-button.js

```diff
@@ -3,10 +3,10 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Icon from '../icon';
 import Button from '../button';
-import SplitButton from '../split-button';
+// import SplitButton from '../split-button';
 import './multi-action-button.scss';
 
-class MultiActionButton extends SplitButton {
+class MultiActionButton extends React.Component {
```

Comment:
`import React, { Component } from 'react'`
and then 
`class MultiActionButton extends Component { ...`
What do you think about this?

Reply:
Can do, the `MultiAction` files are to be done in a different PR I just changed them as it was causing issues in storybook